tests/conftest.py

```diff
@@ -153,29 +153,45 @@ def ceremony_inputs() -> Tuple[List[str], List[str], List[str], List[str]]:
         "2",  # Please enter root threshold
         f"{_PEMS / 'JC.pub'}",  # Please enter path to public key
         "my rsa key",  # Please enter key name
-        "0",  # Please press 0 to add key, or remove key by entering its index  # noqa
         f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
         "JimiHendrix's Key",  # Please enter key name
-        "0",  # Please press 0 to add key, or remove key by entering its index.  # noqa
         f"{_PEMS / 'JJ.pub'}",  # Please enter path to public key
         "JanisJoplin's Key",  # Please enter key name
-        "1",  # Please press 0 to add key, or remove key by entering its index. Press enter to contiue  # noqa
-        "",  # Please press 0 to add key, or remove key by entering its index. Press enter to contiue  # noqa
     ]
     input_step3 = [  # Configure Online Key
         f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
         "Online Key",  # Please enter a key name
     ]
     input_step4 = [  # Sign Metadata
-        "1",  # Please enter signing key index
         f"{_PEMS / 'JH.ed25519'}",  # Please enter path to encrypted private key  # noqa
-        "1",  # Please enter signing key index
         f"{_PEMS / 'JJ.ecdsa'}",  # Please enter path to encrypted private key  # noqa
     ]
 
     return input_step1, input_step2, input_step3, input_step4
 
 
+@pytest.fixture
+def ceremony_selection(monkeypatch) -> lambda *a: str:
+    # public key selection options
+    selection_options = iter(
+        (
+            # adding/removing root public signing keys
```

Comment:
Besides this comment probably be worth it to map it to `input_steps` from `ceremony_inputs` like that:
```suggestion
            # adding/removing root public signing keys (selections for input_step2)
```